koku/api/report/test/ocp/test_ocp_query_handler.py

```diff
@@ -1469,3 +1469,20 @@ def test_multi_exclude_functionality(self):
                         self.assertIsNotNone(grouping_list)
                         for group_dict in grouping_list:
                             self.assertNotIn(group_dict.get(ex_opt), [exclude_one, exclude_two])
+
+    def test_ocp_cpu_query_group_by_storage_class(self):
+        """Test that group by storageclass functionality works."""
+        group_by_key = "storageclass"
+        url = f"?filter[time_scope_units]=month&filter[time_scope_value]=-1&filter[resolution]=monthly&filter[limit]=3&group_by[{group_by_key}]=*"  # noqa: E501
+        query_params = self.mocked_query_params(url, OCPCpuView)
+        handler = OCPReportQueryHandler(query_params)
+        query_data = handler.execute_query()
+        for data in query_data.get("data"):
```

Comment:
We need to assert that there is data, otherwise this test will pass if there is no data.